expression/builtin_string.go

```diff
@@ -2916,45 +2916,53 @@ func (b *builtinOrdSig) Clone() builtinFunc {
 // See https://dev.mysql.com/doc/refman/5.7/en/string-functions.html#function_ord
 func (b *builtinOrdSig) evalInt(row chunk.Row) (int64, bool, error) {
 	str, isNull, err := b.args[0].EvalString(b.ctx, row)
+	strBytes := hack.Slice(str)
```

Comment:
I think putting it before https://github.com/pingcap/tidb/pull/29527/files#diff-159c25a0ed92223cdf33859a7b1d5d25d7d018136ba44c4eabdac0d1619d2138R2931 is better. We can avoid unnecessary calls in some cases

Reply:
Resolved.